py/desitarget/gfa.py

```diff
@@ -413,6 +448,10 @@ def select_gfas(infiles, maglim=18, numproc=4, cmx=False):
         Magnitude limit for GFAs in Gaia G-band.
     numproc : :class:`int`, optional, defaults to 4
         The number of parallel processes to use.
+    tilesfile : :class:`str`, optional, defaults to ``None``
+        Name of tiles file to load. If ``None`` or file doesn't exist,
+        then load the entire footprint. If no path to the file is
+        sent, then look in $DESIMODEL/data/footprint.
```

Comment:
This description isn't right.  tilesfiles is passed to `desimodel.io.load_tiles`, which has the following logic:
* if None: load the full footprint
* if no path:
  * if a file of that name is both local and in $DESIMODEL/data/footprint, print a warning and use the desimodel version
  * if only in $DESIMODEL/data/footprint or only local, use that
  * if in neither, raise an error
* with path: load it if it exists, or raise an exception if it doesn't.

For the purposes of this docstring, I think it's ok to just say this is passed to `desimodel.io.load_tiles()` and let that code document the subtleties.

Reply:
Noted. Thanks for the clarification. I changed the docstring to simply point to the description in `desimodel.io.load_tiles()`.